src/index.scss

```diff
@@ -5,6 +5,7 @@ $modal-max-width: 650px;
 @import "~@edx/brand/paragon/variables";
 @import "~@edx/paragon/scss/core/core";
 @import "~@edx/brand/paragon/overrides";
+@import "~@edx/frontend-enterprise/dist/frontend-enterprise";
```

Comment:
interesting, this import pattern, would have been nice to get something like `~@edx/frontend-enterprise/styles` but okay...

Reply:
Yeah, I don't love it, tbh, but we have a ticket to update the build to Webpack and hopefully we can change it then.